src/Microsoft.AspNet.Mvc.Razor/RazorTextWriter.cs

```diff
@@ -65,7 +70,7 @@ public override void Write(char value)
         /// <inheritdoc />
         public override void Write(object value)
         {
-            var htmlString = value as HtmlString;
+            var htmlString = value as IHtmlContent;
```

Comment:
... but please fix the variable name to avoid misrepresenting things. It's not an `HtmlString`.